### DISCUSSION TITLE
How prevent overwriting a file when performing --upgrade?

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#1043] [Q&A] [answered]
Anytime I do an `--upgrade` my custom notification.mp3 gets overwritten.

Is there a way to prevent that from happening?

**Comment by vladmandic** [comment]
Don't call it notification.mp3, call it ANYTHING ELSE.

**Reply by brknsoul** [reply, marked as answer]
Answer: Give the custom mp3 a different name, and point to it in Settings > Live Previews > Path to notification sound.
